src/qt/splashscreen.h

```diff
@@ -36,8 +36,8 @@ public Q_SLOTS:
     /** Show message and progress */
     void showMessage(const QString &message, int alignment, const QColor &color);
 
-    /** Sets the break action */
-    void setBreakAction(const std::function<void(void)> &action);
+    /** Sets if cancel is currently possible */
```

Comment:
I'd consider replacing "cancel possible" with "shutdown possible" everywhere in this PR. The function this is controlling is called `StartShutdown`. The display text is `press q to shutdown`. Why should this thing be named differently?